TADashboard/src/main/java/page_objects/DataProfilesDisplayFieldsPage.java

```diff
@@ -0,0 +1,23 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
+import utils.common.Constants;
+
+import java.util.List;
+
+public class DataProfilesDisplayFieldsPage extends BasePage {
+    private Label lblProfileHeader = new Label("css=#profilesettings > tbody > tr > td.profilesettingheader");
+    private Label lblDisplayFieldsLabels = new Label("css=#profilesettings > tbody label");
+    private CheckBox dynamicCheckBoxOfField = new CheckBox("xpath=//table[@id='profilesettings']/tbody//label[contains(text(),'%s')]/input");
```

Comment:
this is not a good name. show me a screenshot of this element.

Reply:
![image](https://user-images.githubusercontent.com/56109274/134447734-acc95d93-faa1-40c5-880b-d842387091de.png)

Reply:
Please check again